concent_api/core/tests/utils.py

```diff
@@ -169,22 +186,23 @@ def _get_deserialized_report_computed_task(
     def _get_deserialized_task_to_compute(
         self,
         timestamp: Union[str, datetime.datetime, None] = None,
-        deadline = None,
+        deadline: Union[str, int, None] = None,
```

Comment:
Why not Optional here ?

Reply:
@dybi suggested to do it like this...

Reply:
@dybi What is the reason for doing it this way but only in few selected places ?

Reply:
`Optional[Type]` is the same as `Union[Type, None]`; for me having `Union[str, int, None]` is more readable than `Optional[Union[str, int]]`, in other words, when I have `Union` of `None` and more that one type, I prefer to list type explicitly in `Union`